Matrix.py

```diff
@@ -0,0 +1,123 @@
+class Matrix:
+    def __init__(self, x):
```

Comment:
Rename <x>, use name that have some meaning about what is x.

Reply:
changed to "data"